README.md

```diff
@@ -188,16 +184,15 @@ Alternatively add symlinks in your `.git/hooks` directory to any of the githooks
 
 ## Policy on Altcoins/Altchains
 
-Patches which add support for non-Bitcoin cryptocurrencies by adding constants
-to existing enums (e.g. to set the network message magic-byte sequence) are
-welcome. Anything more involved will be considered on a case-by-case basis,
-as the altcoin landscape includes projects which [frequently appear and
-disappear, and are poorly designed anyway](https://download.wpsoftware.net/bitcoin/alts.pdf)
-and keeping the codebase maintainable is a large priority.
+Patches which add support for non-Bitcoin cryptocurrencies by adding constants to existing enums
+(e.g. to set the network message magic-byte sequence) are welcome. Anything more involved will be
+considered on a case-by-case basis, as the altcoin landscape includes projects which [frequently
+appear and disappear, and are poorly designed
+anyway](https://download.wpsoftware.net/bitcoin/alts.pdf) and keeping the codebase maintainable is a
+large priority.
```

Comment:
Given we don't have such enums (anymore?) we should just drop this and say we don't support shitcoins. `ChainHash` is closest to supporting shitcoins, probably not worth the trouble.

Reply:
Yeah, I think so. The altcoin landscape has changed dramatically since 2014 anyway. This was written back when most shitcoins were clonecoins and "supporting" them was a matter of tweaking a couple constants. Nowadays the most popular things are all pretty unique. Even litecoin has Confidential Transactions.

Reply:
Removed as an additional patch.